### PR TITLE
fix(form): vertically align switches and checkboxes

### PR DESCRIPTION
**Pull request by maltja**
fix Lundalogik/crm-feature#1629

## Review:
- [x] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [x] Commits have the correct *type* for the changes made
- [x] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [x] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [x] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [x] iOS

### CODE REVIEW

**Comment by Kiarokh** [comment]
@maltja I have pushed a few more changes, and a fixup to your original commit. I think someone else should review this now, since I have been making changes too

File: src/components/form/form.scss [hunk 1]
```diff
@@ -111,3 +111,13 @@
         grid-column: 1/-1;
     }
 }
+
+.form-group {
+    limel-checkbox,
+    limel-switch {
+        margin-left: pxToRem(8);
+        min-height: calc(
+            var(--min-height-of-one-row) - #{pxToRem(15)} /*height of helper text*/
```
Comment (adrianschmidt) [inline src/components/form/form.scss:120]: If we don't have a variable for "height of helper text", maybe we should add one? 🙂

File: src/components/switch/switch.scss [hunk 2]
```diff
@@ -2,8 +2,14 @@
...
 @import '@limetech/mdc-switch/mdc-switch';
 
+:host {
+    display: inline-flex;
+    align-items: center;
+}
+
 .mdc-switch {
     @include mdc-switch-toggled-on-color(primary);
+    line-height: normal;
```
Comment (adrianschmidt) [inline src/components/switch/switch.scss:12]: Was this intended to go on the label? I've only checked in Chrome on macOS, but switching this on and off makes no difference, at least not where I'm checking. I tried adding `line-height:normal` to the label though, and that changed the alignment a pixel or two, the result of which I thought looked better.

**Review by adrianschmidt — COMMENTED** [review]
Very nice! I left two small comments.

File: src/components/form/form.scss (hunk 1 above)
Comment (Kiarokh) [reply src/components/form/form.scss:120]: great Idea. I'll do a fixup

File: src/components/switch/switch.scss (hunk 2 above)
Comment (Kiarokh) [reply src/components/switch/switch.scss]: awesome catch! yes, it was for the label 😅

**Comment by Kiarokh** [comment]
Thanks @adrianschmidt I pushed two fixups

**Review by adrianschmidt — APPROVED** [review]

**Pull request merged**